app/styles/theme.scss

```diff
@@ -1,3 +1,4 @@
+$Theme-margin-xSmall: 13px;
```

Comment:
Value was initially worth 15px, changed to the specific spacings for footer nav links as seen on zeplin;
This is related to the above comment on footer nav link margins, r76207969